src/code/z_play.c

```diff
@@ -1265,10 +1265,10 @@ void Play_DrawMain(PlayState* this) {
                     if (this->skyboxId && !this->envCtx.skyboxDisabled) {
                         if ((this->skyboxId == 1) || (this->skyboxId == 3)) {
```

Comment:
```suggestion
                        if ((this->skyboxId == SKYBOX_NORMAL_SKY) || (this->skyboxId == 3)) {
```
z_play could use a sweep of `skyboxId`